src/main/java/graphql/GraphqlErrorHelper.java

```diff
@@ -73,7 +73,10 @@ public static Object location(SourceLocation location) {
         if (line < 1 || column < 1) {
             return null;
         }
-        return Map.of("line", line, "column", column);
+        LinkedHashMap<String, Object> map = new LinkedHashMap<>(2);
```

Comment:
Don't we have a Util method for that? Maybe not, but we should have that emulates the Map.of ...